api/v1/types.go

```diff
@@ -0,0 +1,24 @@
+package v1
+
+// RuntimeTransactionList is the API response for RuntimeListTransactions.
+type RuntimeTransactionList struct {
+	Transactions []RuntimeTransaction `json:"transactions"`
+}
+
+// RuntimeTransaction is the API response for RuntimeGetTransaction.
+type RuntimeTransaction struct {
+	Round   int64   `json:"round"`
+	Index   int64   `json:"index"`
+	Hash    string  `json:"hash"`
+	EthHash *string `json:"eth_hash"`
+	// TODO: timestamp
+	Sender0   string  `json:"sender_0"`
```

Comment:
This and `To` are oasis-derived-from-eth-address addresses, right? We should expose the underlying eth address here (and in the  Explorer) too. Depending on how people onboard, the might think primarily or even only in terms of eth addresses. (Think Opal, say no more say no more this is a public repo, yadda yadda)

Reply:
ya people will probably want the eth address too